providers/apis/polymarket/README.md

```diff
@@ -0,0 +1,86 @@
+# PolyMarket Provider
+
+Docs: https://docs.polymarket.com/
+
+Polymarket is a web3 based prediction market. This provider uses the Polymarket CLOB REST API to access the off-chain order book. 
+
+## How it Works
+
+Polymarket uses [conditional outcome tokens](https://docs.gnosis.io/conditionaltokens/), a token that represents an outcome of a specific event. All tokens in Polymarket are denominated in terms of USD.
+
+Tickers take the form of the `<market_slug>?<outcome>/USD`:
+
+Example: `WILL-BERNIE-SANDERS-WIN-THE-2024-US-PRESIDENTIAL-ELECTION?YES/USD`
+
+The offchain ticker is expected to be _just_ the token_id.
+
+example: `95128817762909535143571435260705470642391662537976312011260538371392879420759`
+
+The Provider simply calls the `/price` endpoint of the CLOB API. There are two query parameters:
+
+* token_id
+* side
+
+Side can be either `buy` or `sell`. For this provider, we hardcode the side to `buy`.
+
+## Market Config
+
+Below is an example of a market config for a single Polymarket token.
+
+```json
+ {
+  "markets": {
+    "WILL-BERNIE-SANDERS-WIN-THE-2024-US-PRESIDENTIAL-ELECTION?YES/USD": {
+      "ticker": {
+        "currency_pair": {
+          "Base": "WILL-BERNIE-SANDERS-WIN-THE-2024-US-PRESIDENTIAL-ELECTION?YES",
+          "Quote": "USD"
+        },
+        "decimals": 3,
+        "min_provider_count": 1,
+        "enabled": true
+      },
+      "provider_configs": [
+        {
+          "name": "polymarket_api",
+          "off_chain_ticker": "95128817762909535143571435260705470642391662537976312011260538371392879420759"
```

Comment:
How do you get this value?

Reply:
you can get this from the API, `curl https://clob.polymarket.com/markets | jq .` 

OR

what i did was just go to the polymarket website itself and inspect element on the page and snipe it from the webapps network calls lol